packages/components/src/Tooltip/Tooltip.stories.tsx

```diff
@@ -100,13 +81,22 @@ export const LongButtonText: StoryObj<Args> = {
 };
 
 export const Interactive = {
-  ...LightVariant,
   args: {
     visible: undefined,
+    children: (
+      <Button className="mx-32 my-32">
+        Tooltip trigger with hover done by storybook. <br />
+        Click within canvas to make interactive.
```

Comment:
Is this specific to this repo? I don't have to click into the canvas in `traject` to get the hover to work. (You do need to have focus on that browser window though.)

Reply:
The story loads and immediately hovers the button to spawn to the tooltip, so it doesn't look interactive. Have to click  to despawn the tooltip created by the play function.

How should I word this to make such message come across?

Reply:
Ah, I see. So you have to click to dismiss the tooltip, and then you can hover over it?

Reply:
Yup. Hovering and after hover also despawns it, but basically have to interact with it a little

Reply:
I think this is fine as-is, but I'll throw out a suggestion for your consideration.

I'm wondering if non-devs will understand the "canvas" part here, so I was thinking you could move the text out of the button and change it to "Click somewhere in this area to dismiss the tooltip, then hover over the button to make it reappear." and then you could change the button text to something like "Hover here to see tooltip after clicking somewhere outside".

I'm probably overthinking this, so don't spend too much time dwelling on it. Thanks for figuring out how to make this story work! ❤️

Reply:
Great point 👍 

I think devs will probably be confused by this as well, since (I believe) "canvas" here refers to the Storybook "canvas", not an actual HTML canvas.

Most folks probably won't be that familiar with that terminology.

Reply:
Updated story as advised, thanks!